src/components/15Years/15YearsIndividualsData.js

```diff
@@ -0,0 +1,247 @@
+		</p><p>
+		Although Howard fell just short of his initial lending aspirations, he has created life-changing impact for thousands of Kiva borrowers over the past 12 years. Kiva’s community of generous lenders, like Howard, is fundamental to building the financially inclusive world we envision.
+		</p><p>
+		"[Kiva] allows me to create the world that I want it to be, in a small way. That’s all because I happened to pick up the paper that day with my coffee." We’re so happy you did, Howard.
+		</p>
+		`,
+		videoId: null,
+		cta1: 'Lend like Howard',
+		cta2: 'Lend monthly',
+		link1: '/lend-by-category',
+		link2: '/monthlygood',
+	},
+	{
+		name: 'Julia',
+		title: 'High School Hero',
+		img: 'need-content',
+		body: null,
+		videoId: 174878843,
+		cta1: 'Lend like Julia',
+		cta2: 'Lend monthly to students',
+		link1: '/lend-by-category/education',
+		link2: '/monthlygood?category=education',
+	},
+	{
+		name: 'Drew + Allie',
+		title: 'Lending Power Couple',
+		img: 'drew-allie',
+		body: `<p>
+		Need content
+		</p>
+		`,
+		videoId: null,
+		cta1: 'Lend like Drew and Alli',
+		cta2: 'Lend monthly',
+		link1: '/lend-by-category',
+		link2: '/monthlygood',
+	},
+	{
+		name: 'Apollo Ag',
+		title: 'Small Farm Funder',
+		img: 'need-content',
+		body: null,
+		videoId: 174878843,
+		cta1: 'Support groups like Apollo Agriculture',
+		cta2: 'Support groups like Apollo Agriculture',
+		link1: 'Africa filtered loans', // TODO: find url.
+		link2: 'Africa filtered loans', // TODO: find url.
+	},
+	{
+		name: 'WWBIC',
+		title: 'Heartland Supporters',
+		img: 'need-content',
+		body: null,
+		videoId: 174878843,
+		cta1: 'Lend to U.S. small businesses',
+		cta2: 'Lend monthly to American entrepreneurs',
+		link1: '/lend-by-category/kiva-u-s',
+		link2: '/monthlygood?category=us_borrowers',
+	},
+	{
+		name: 'David Kitusa',
+		title: 'The Local Changemaker',
+		img: 'need-content',
+		body: `<p>
+		Need content
+		</p>
+		`,
+		videoId: null,
+		cta1: 'Lend to African borrowers',
+		cta2: 'Lend to African borrowers',
+		link1: 'Africa filtered loans', // TODO: find url.
+		link2: 'Africa filtered loans', // TODO: find url.
+	},
+	{
+		name: 'Rachel Chang',
+		title: 'Protocol Practitioner',
+		img: 'rachel',
+		body: `<p>
+		Need content
+		</p>
+		`,
+		videoId: null,
+		cta1: 'Lend to African borrowers',
+		cta2: 'Lend to African borrowers',
+		link1: 'Africa filtered loans', // TODO: find url.
+		link2: 'Africa filtered loans', // TODO: find url.
```

Comment:
@mcstover Friendly surfacing that we'll need to get a game plan for these types of links